src/Shared/ARCtrl.Helper.fs

```diff
@@ -207,7 +197,7 @@ module Extensions =
         member this.GetHeader(columnIndex: int) =
             DataMap.getHeader(columnIndex)
 
-        static member ColumnCount = 9
+        static member ColumnCount = 6
```

Comment:
Why do we know the columnCount?

Reply:
Because this worksheet does always consist of the exactly same columns: https://github.com/nfdi4plants/ARC-specification/blob/main/ISA-XLSX.md#datamap-table-sheets

I create a similiar syntax to ArcTable to allow reusing the ArcTable ui cells and functions.

Reply:
k, maybe you can add a comment. I was just wondering because it used to be 9.